molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -75,13 +76,6 @@
         <script src="https://apis.google.com/js/platform.js<#if authenticated?? && authenticated>?onload=onLoad</#if>"
                 async defer></script>
     </#if>
-
-    <!--[if IE 9]>
-    <#-- used to disable the genomebrowser in IE9 -->
-        <script>top.molgenis.ie9 = true;</script>
-        <#-- required by dalliance-compiled.js to load the genomebrowsers in IE9 -->
-        <script src="<@resource_href "/js/typedarray.min.js"/>  </script>
-    <![endif]-->
```

Comment:
Step 1: disable genome browser for IE9
Step 2: load js to be able to load genome browser for IE9

Brilliant stuff 🤣

Reply:
;-)

The code was moved to view-dataexplorer-mod-data.ftl without the typedarray script tag